symengine/polys.h

```diff
@@ -0,0 +1,70 @@
+/**
+ *  \file polys.h
+ *  Class for sparse Polynomial: Polynomial
+ *
+ **/
+#ifndef SYMENGINE_POLYS_H
+#define SYMENGINE_POLYS_H
+
+#include <symengine/basic.h>
+#include <symengine/dict.h>
+
+namespace SymEngine {
+//! Polynomial Class
+class Polynomial : public Basic{
+public:
+    //! `vars_` : Variables of the Polynomial
+    //! `polys_set_` : holds the Polynomial
+    vec_symbol vars_;
+    hash_set polys_set_;
```

Comment:
@bluescarni wouldn't the compiler check that anyway once the .cpp file (which includes piranha) is compiled?